### PR TITLE
DCOS-21723: use metadata instead of calculation

### PR DESCRIPTION
**Pull request by nLight**
**TL;DR** This PR removes all the resources aggregation logic and replaces it with the `used_resources` for Frameworks.

**Tell me more**
Looks like we've mistakenly assumed that allocated resources for Frameworks should be aggregated from their tasks as with Marathon Services. Turned out that there's a special metadata on the Framework protobuf that has `used_resources` information.

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [x] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

Closes DCOS-21723

### CODE REVIEW

**Comment by nLight** [comment]
Anecdotally I've fixed the issue once already https://github.com/dcos/dcos-ui/commit/49b2308bcabec8e79b0ecced4c38e5118018970b :D

**Comment by nLight** [comment]
Checking if the solution is applicable for 1.11, looks like it is not

File: plugins/services/src/js/containers/services/ServicesContainer.js [hunk 1]
```diff
@@ -144,10 +143,6 @@ class ServicesContainer extends React.Component {
   componentDidMount() {
     DCOSStore.addChangeListener(DCOS_CHANGE, this.onStoreChange);
 
-    // Don't block the whole screen if Mesos state is not there
```
Comment (jaxesn) [inline plugins/services/src/js/containers/services/ServicesContainer.js]: This seems unrelated?

File: plugins/services/src/js/structs/Framework.js [hunk 2]
```diff
@@ -93,45 +93,10 @@ module.exports = class Framework extends Application {
...
-        };
-      }, frameworkResources);
-  }
-
-  getUsedResources() {
+    // There's an unfortunate naming issue in Mesos.
+    // used_resources is actually allocated resources
+    // the name can't be changed to keep backward compatibility.
+    // This is why `getResources` returns `used_resources`
     return (
```
Comment (jaxesn) [inline plugins/services/src/js/structs/Framework.js]: Well thats easier...

File: src/js/stores/DCOSStore.js [hunk 3]
```diff
@@ -440,7 +440,7 @@ class DCOSStore extends EventEmitter {
       .getServiceList()
       .reduceItems(function(memo, framework) {
         if (framework instanceof Item) {
-          memo[framework.get("name")] = framework.get();
+          memo[`/${framework.get("name")}`] = framework.get();
```
Comment (jaxesn) [inline src/js/stores/DCOSStore.js]: Is this just a cleanup, or was the `/` missing and required somewhere else?

Comment (nLight) [reply src/js/stores/DCOSStore.js]: Have a look at the change down below. `/` was required to make them equal

File: plugins/services/src/js/containers/services/ServicesContainer.js (hunk 1 above)
Comment (nLight) [reply plugins/services/src/js/containers/services/ServicesContainer.js]: I looked back to the commit where I added that `getResources` method back then. There I added this event listener. It was required since we haven't had stream and tended to use `MesosStateStore` to calculate resources as use can see from the method I deleted

**Comment by nLight** [comment]
@fabs that's funny. Those aren't frameworks are they? You can always compare with `/mesos` UI

**Comment by Fabs** [comment]
@nLight I think they are the simplest framework, the hello-world. I deployed them from the catalog, and they have 3 tasks, a hello-world (which I suppose is the framework) and one `hello` and one `world`.  I left them running in the SF cluster.

**Comment by Fabs** [comment]
Hum, as a starter, mesos know all tasks:
image

**Comment by Fabs** [comment]
Yeah, your values match what mesos know.

image

**Comment by Fabs** [comment]
@orlandohohmeier @nLight I think this will need a communication to our users that will after update just see that the values they are used to see changes. I think that should be very well explained in CHANGELOG.

**Comment by nLight** [comment]
Making sense of data: 
services table show allocated resources.
Task detail page shows used resources
allocated resources are the resources that Mesos gave to the framework. Framework can use less resources that has been allocated. In this scenario 2 of 3 tasks use 25 Mb disk which sums up to 50 meg. They also use less RAM which is ok because Task

**Comment by nLight** [comment]
We could add a tooltip to the Services Table indicating that resources are allocated resources

**Comment by Fabs** [comment]
@nLight hum. I think that could help. But I also think it is something that needs more direct comunication. like:

BREAKING CHANGES - The ui change this way to count resources, to this way to count resources, but only on this particular area. Here is the blogs.dcos.happyusers/how-the-dcos-ui-count-resources. 

And then have this amazing text explaining our cluster operators what changed.

**Comment by nLight** [comment]
I mean, that would be rather short amazing embarrassing text 😆 Like we used to sum all the tasks resources now we just show what Mesos gives us :D

**Comment by nLight** [comment]
https://jira.mesosphere.com/browse/DCOS-39095
